src/FormCollection/Form/UXCollectionType.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\FormCollection\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @final
+ * @experimental
+ */
+class UXCollectionType extends AbstractType
+{
+    public function getParent()
+    {
+        return CollectionType::class;
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $defaultButtonAddOptions = [
+            'label' => 'Add',
+            'class' => '',
+        ];
+        $defaultButtonDeleteOptions = [
+            'label' => 'Remove',
+            'class' => '',
+        ];
```

Comment:
I'd also like a `label_html` option which is set to a boolean (defaults to false). If true, we would render the label "not escaped" (i.e. raw). Basically, we're trying to accomplish this: https://symfony.com/blog/new-in-symfony-5-1-form-improvements#allow-html-contents-in-form-labels

This would allow, for example, FontAwesome icons to be used with the buttons.